examples/knative/powershell/kn-ps-webhook/handler.ps1

```diff
@@ -0,0 +1,123 @@
+Function Process-Init {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Init`n"
+
+   Write-Host "$(Get-Date) - Init Processing Completed`n"
+}
+
+Function Process-Shutdown {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Shutdown`n"
+
+   Write-Host "$(Get-Date) - Shutdown Processing Completed`n"
+}
+
+Function Process-Handler {
+   [CmdletBinding()]
+   param(
+      [Parameter(Position=0,Mandatory=$true)]$Headers,
+      [Parameter(Position=1,Mandatory=$true)]$Context
+   )
+
+   Import-Module CloudEvents.Sdk
```

Comment:
Can this moved outside of any of the functions? I feel this is only needed once when sourcing the handler.ps?